.github/workflows/e2e.yml

```diff
@@ -0,0 +1,68 @@
+
+name: e2e
+
+# triggered by labeled PR
+on:
+  pull_request_target:
+    types: [ labeled ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    if: ${{ github.event.label.name == 'e2e' }}
+    strategy:
+      matrix:
+        # os: [ubuntu-latest, ubuntu-24.04-arm]
+        os: [ubuntu-latest]
+        # distro: [fedora-41, fedora-42, fedora-r43, centos-stream-9, centos-stream-10]
+        distro: [centos-stream-10]
+    runs-on: ${{ matrix.os }}
+
+    # Required to push container image to ghcr.io
+    # https://github.com/orgs/community/discussions/57724
+    permissions:
+      contents: read
+      packages: write
```

Comment:
Per discussion let's create a default flow in Github Actions that builds a bootc container image *tagged with the sha digest* and pushes to `ghcr.io` under the repo namespace.

Then we can run further tests on that container image.

I think we'll need a scheduled workflow to prune the images though.